src/main/java/com/team701/buddymatcher/controllers/timetable/TimetableController.java

```diff
@@ -14,6 +14,7 @@
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
```

Comment:
It seem that one case of more importing that necessary was overlooked

Reply:
Changes have been made.